RLTest/__main__.py

```diff
@@ -385,12 +387,16 @@ def watcher_thread(self):
                 print(Colors.Bred("Failed on timeout function, %s" % str(e)))
             os._exit(1)
 
+    def reset(self):
+        self.timeout_time = time.time() + self.timeout
```

Comment:
shouldn't it acquire the condition while making this change?

Reply:
Python has a GIL anyway so it will not cause the other thread not to see the update. The worst that can happened is that there will be a context switch before the timeout_time was set, in this case we can consider it as not been set yet.